src/common/context/__tests__/GlobalContext.test.tsx

```diff
@@ -1,6 +1,7 @@
 import { render, screen, waitFor } from '@testing-library/react';
 import { useSearchParams as useSearchParamsActual } from 'next/navigation';
 import { useContext } from 'react';
+import { CookiesProvider } from 'react-cookie';
```

Comment:
why are we adding this?

Reply:
it's needed now after the upgrade, more info in migration guide.